### ISSUE TITLE
error "No module named contact"

### ISSUE THREAD

**Issue by leberknecht** [Maufeat/bs.to-kodi-repo#1]
Hi, i'm getting
```
Error Type: <type 'exceptions.ImportError'>
                                            Error Contents: No module named contact
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.burningseries/main.py", line 6, in <module>
```
What module do i have to install to fix it?

**Comment by vbprofi** [comment]
You need an User-Account. You have register an new Account at bs.to and insert the logindata in this addon.

**Comment by leberknecht** [comment]
Hm, i'm pretty sure i had setted up the login..but meanwhile i'm getting a different error, indeed. Could that be related to bs.to changing API keys? (a rumor i found here: https://board.bs.to/index.php?/topic/1989-api/ )

**Comment by vbprofi** [comment]
yes, bs.to had changed the API-Key. if you get other error than above, that is meening, login is failed. bs.to doesn't give new API-Key. I try with livestreams pro to create a working solution.
